spec/spec_helper.rb

```diff
@@ -45,6 +91,22 @@
     mocks.verify_partial_doubles = true
   end
 
+  config.before(:suite) do
+    DatabaseCleaner.strategy = :transaction
+    DatabaseCleaner.clean_with(:truncation)
+    start_es_server unless ENV['ES_SKIP']
```

Comment:
what about doing this before specific specs rather than before the whole test suite? (wondering if there will be problems trying to run individual specs that require the es_server if we only call this here)